.github/workflows/scripts/e2e-verify.common.sh

```diff
@@ -115,7 +115,7 @@ e2e_verify_decoded_token() {
     # GitHub's information.
     _e2e_verify_query "$decoded_token" "$GITHUB_ACTOR_ID" '.github.actor_id'
     _e2e_verify_query "$decoded_token" "$GITHUB_EVENT_NAME" '.github.event_name'
-    _e2e_verify_query "$decoded_token" "$GITHUB_JOB" '.github.job'
+    _e2e_verify_query "$decoded_token" "setup-token" '.github.job'
```

Comment:
I wonder if we should even record `$GITHUB_JOB` if it's going to be different than the build job. Maybe make an issue for this so we can follow up?

Reply:
I removed the job from the token. Successful run https://github.com/laurentsimon/slsa-github-generator/actions/runs/4429655425

Reply:
Looks like the unit tests for `verify-token` need to be updated to remove the job id.

Reply:
They are removed in the latest commit.

Reply:
my bad, let me fix this!